### ISSUE TITLE
Connect it to MediaPLayer

### ISSUE THREAD

**Issue by salvonos** [Yalantis/Horizon#17]
Is there a way to user the visualiser reacting to a MediaPlayer?

**Comment by ffournier** [comment]
Yes you can by use the Visualizer of the API Android https://developer.android.com/reference/android/media/audiofx/Visualizer.html. YOu can get the buffer by the method setOnCaptureListener

**Comment by bliveinhack** [comment]
Hello,

Can you guys please upload sample with media player. it would be nice to see with music player too.
